login-app/src/actions/index.js

```diff
@@ -0,0 +1,33 @@
+import axios from "axios"
+
+export const LOGIN_START = "LOGIN_START"
+export const FETCH_DATA_START = "FETCH_DATA_START"
+export const FETCH_DATA_SUCCESS = "FETCH_DATA_SUCCESS"
+export const FETCH_DATA_FAILURE = "FETCH_DATA_FAILURE"
+
+export const login = creds => dispatch => {
+  dispatch({ type: LOGIN_START })
+
+  return axios
+    .post('http://localhost:5000/api/login', creds)
+    .then(res => localStorage.setItem("token", res.data.payload))
+}
```

Comment:
You have a login start action, but you don't have a login success or failure action.  Consequently, you will only be in the starting state